releasenotes/notes/qdrift-as-product-formula-044a37a106a45a47.yaml

```diff
@@ -0,0 +1,6 @@
+---
+prelude: >
+    Reformulate the former QDrift class in qiskit.opflow.evolutions.trotterizations as a ProductFormula instead of a TrotterizationBase.
```

Comment:
This is only for the release prelude, not for the PRs afaik 🙂 
```suggestion
```

Reply:
This is correct - the prelude goes to the top of the whole release to become like its summary.

Reply:
removed the prelude and rewrote a bit the whole release note